### PR TITLE
BA-1097-Django-Cloudflare-Stream-Package-to-BA-BE

### PR DESCRIPTION
**Pull request by MattGalvis**
These didnt have any tests when they were created

### CODE REVIEW

File: baseapp-cloudflare-stream-field/testproject/__init__.py
Comment (pablogadhi) [inline baseapp-cloudflare-stream-field/testproject/__init__.py:1]: Aren't we missing the wsgi configuration for the testproject @MattGalvis ? Could run it just fine after adding it.

File: baseapp-cloudflare-stream-field/testproject/wsgi.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"""
```
Comment (Janekk) [inline baseapp-cloudflare-stream-field/testproject/wsgi.py:1]: can we delete bec there is such file on `baseapp-core`?

File: baseapp-cloudflare-stream-field/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+[build-system]
```
Comment (Janekk) [inline baseapp-cloudflare-stream-field/pyproject.toml:1]:
```
[tool.black]
...
[tool.isort]
...
```
sections are already in the root dir `baseapp-backend/pyproject.toml` and CI uses that one, so maybe just leave `[build-system]` section?

**Review by Janekk — CHANGES_REQUESTED** [review]

**Review by dpTsl — APPROVED** [review]

**Comment by dpTsl** [comment]
Hey, Matt! That is not related to your PR but could you please rename the jobs to follow a pattern?
I think we can remove "workflow" from [baseapp-core-workflow](https://github.com/silverlogic/baseapp-backend/pull/14/files#diff-d8485ae5feb85d5310fbb669eca158cacafdba09b695a16d47720b69b9072e23R10)", from [baseapp-auth-workflow](https://github.com/silverlogic/baseapp-backend/pull/14/files#diff-d8485ae5feb85d5310fbb669eca158cacafdba09b695a16d47720b69b9072e23R14) and also from the [comment](https://github.com/silverlogic/baseapp-backend/pull/14/files#diff-d8485ae5feb85d5310fbb669eca158cacafdba09b695a16d47720b69b9072e23R38).

And I think that we could also remove the [baseapp-reactions comment](https://github.com/silverlogic/baseapp-backend/pull/14/files#diff-d8485ae5feb85d5310fbb669eca158cacafdba09b695a16d47720b69b9072e23R22)

**Review by Janekk — APPROVED** [review]

**Pull request merged**